### ISSUE TITLE
Hash mismatch

### ISSUE THREAD

**Issue by excitoon** [bazelbuild/bazel-skylib#164]
```
[ex@cube ~]$ curl -L https://github.com/bazelbuild/bazel-skylib/releases/download/0.8.0/bazel-skylib.0.8.0.tar.gz | shasum -a 256
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   616    0   616    0     0   1555      0 --:--:-- --:--:-- --:--:--  1551
100 72941  100 72941    0     0  59543      0  0:00:01  0:00:01 --:--:-- 59543
2ef429f5d7ce7111263289644d233707dba35e39696377ebab8b0bc701f7818e  -
[ex@cube ~]$ curl https://mirror.bazel.build/github.com/bazelbuild/bazel-skylib/archive/0.8.0.tar.gz | shasum -a 256
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 73708  100 73708    0     0   278k      0 --:--:-- --:--:-- --:--:--  278k
2ea8a5ed2b448baf4a6855d3ce049c4c452a6470b1efd1504fdb7c1c134d220a  -
```

**Comment by laurentlb** [comment]
They are two different files. If you look at the content, you'll see that they include the same set of files, but one archive has a parent directory.

These two files are the same:
* https://github.com/bazelbuild/bazel-skylib/archive/0.8.0.tar.gz
* https://mirror.bazel.build/github.com/bazelbuild/bazel-skylib/archive/0.8.0.tar.gz

These two files are also the same:
* https://github.com/bazelbuild/bazel-skylib/releases/download/0.8.0/bazel-skylib.0.8.0.tar.gz
* https://mirror.bazel.build/github.com/bazelbuild/bazel-skylib/releases/download/0.8.0/bazel-skylib.0.8.0.tar.gz

**Comment by laurentlb** [comment]
Looks like a mistake in rules_cc

**Comment by excitoon** [comment]
@laurentlb Yeah, once I got an error I went to skylib releases and checked hash of 0.8.0 but missed that url is not the same, and made false assumption that file in mirror is wrong. But really wrong thing was the hash.
Then in bazelbuild/rules_cc#21 @hlopko thought he uploaded wrong file but fixed the hash in ec70973809612f76f0ab4748d7db74da6e5b06b2 .
